src/morale.h

```diff
@@ -76,6 +73,61 @@ enum morale_type : int {
     NUM_MORALE_TYPES
 };
 
+// Morale multiplier
```

Comment:
ACK, if it's too much trouble for this PR and you're planning on more extensive cleanup later, it's fine.